### ISSUE TITLE
[Expert] Ars Nouveau's lightning glyph/spell doesn't work with Lightning Channeling crafts.

### ISSUE THREAD

**Issue by Iris-xii** [NillerMedDild/Enigmatica6#3879]
### Modpack Version

Enigmatica 6 0.5.21+ (commit  047b59fd3c609346212aae3093e213474efbc78b dev branch, expert mode)

### Describe your issue.

Ars Nouveau's lightning glyph/spell doesn't work properly with Lightning Channeling crafts.
To reproduce:

-  Make a Projectile -> Lightning spell
-  Pick a Lightning Channeling recipe, throw the items on the ground 
-  Cast the spell on them and notice that they disappear without dropping any items, unlike when using other sources of lightning bolts such as charged snowballs.

### Crash Report

N/A

### Latest Log

N/A

### Have you modified the modpack?

Yes

### User Modifications

Disabled Alex's mobs crimson mosquito spawns through config, removed Tips and PackMenu

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by Iris-xii** [comment]
The explosion glyph also seems to ignore 'Explosion Crafting' recipes in a similar way

**Comment by theboo** [comment]
that's very weird... i know a few people using the ars fireball loot tome for explosion crafting... but maybe that was just for PNC explosion crafting?

**Comment by Iris-xii** [comment]
Just tried that out, PCR explosion crafting does seem to work with Ars Glyphs

**Comment by MuteTiefling** [comment]
I believe these spells have changed. Checking with Bailey. 

Early on, they were just summoning regular lightning, but that meant he couldn't control the damage. I believe he's changed that to just look like a lightning bolt and deals his damage separately. So it's not a real lightning bolt, sadly

**Comment by MuteTiefling** [comment]
Yep. Seems that they're rolling their own lightning handler now so they can adjust the damage. This would probably require some changes to Interactio to fix. I know @MaxNeedsSnacks is very busy these days, however, so I'm not sure how likely that is. 

For now, the lightning items will have to do.

**Comment by YashTFEM** [comment]
Lightning charges from thermal are always an option
